target/linux/ath79/image/generic-ubnt.mk

```diff
@@ -269,6 +269,15 @@ define Device/ubnt_powerbeam-5ac-gen2
 endef
 TARGET_DEVICES += ubnt_powerbeam-5ac-gen2
 
+define Device/ubnt_powerbridge-m
+  $(Device/ubnt-xm)
+  SOC := ar7241
+  DEVICE_MODEL := PowerBridge-M
+  DEVICE_PACKAGES += rssileds
+  SUPPORTED_DEVICES += powerbridge-m ubnt,powerbridge-m
```

Comment:
This line can be dropped. Was there an image available for ar71xx?

Reply:
ar71xx_bullet-m  is working on the device. Shall I add that?

Reply:
`SUPPORTED_DEVICES += bullet-m`